### PR TITLE
JAMES-3680 LDAP support for James memory server

### PR DESCRIPTION
**Pull request by Arsnael**

### CODE REVIEW

File: server/apps/memory-app/src/main/java/org/apache/james/MemoryJamesConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
+package org.apache.james;
```
Comment (chibenwa) [inline server/apps/memory-app/src/main/java/org/apache/james/MemoryJamesConfiguration.java:1]: License

**Comment by Arsnael** [comment]
I definitely forgot some bindings, don't approve yet!

**Comment by chibenwa** [comment]
> I definitely forgot some bindings, don't approve yet!

Converted to draft to avoid mistakes!

**Comment by Arsnael** [comment]
Now it's ready :)

**Review by chibenwa — APPROVED** [review]

**Comment by chibenwa** [comment]
```
org.apache.james.transport.mailets.AddFooterTest.shouldAddFooterWhenSimpleMessage

Mailet integration tests needs a fix!

[Guice/MissingImplementation]: No implementation for UsersRepository was bound
```

**Pull request merged**